### ISSUE TITLE
Finalize links for footer

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/Advocacy#723]
We need to finalize what links should appear in the footer of the copyright site. The mockup includes:

- Contact Us: https://www.mozilla.org/contact/
- Share This Page: ???
- Join the Conversation: https://twitter.com/Mozilla
- Legal: https://www.mozilla.org/about/legal.html
- Privacy Policy: https://www.mozilla.org/privacy/
- Donate: https://donate.mozilla.org

I think the legal department would also want us to include:
- Cookies: https://www.mozilla.org/privacy/websites/#cookies
- License: https://creativecommons.org/licenses/by-sa/3.0/deed.en

The links above are my best guess. @shaghdoosti, could you please verify and leave a comment below once the links here are the final ones we should use. Do we make any distinctions based on locale - e.g. do we offer different Twitter handles?

**Comment by shaghdoosti** [comment]
I think we should use the same footer for the old site - which I'm assuming have been localized already: 

![screen shot 2017-02-22 at 4 44 53 pm](https://cloud.githubusercontent.com/assets/13626384/23239827/54f11928-f91e-11e6-8089-1cf83cf591f5.png)

**Comment by TheoChevalier** [comment]
It’s fine to reuse the same, but just to clarify, we won’t reuse translations from the old website directly, so if you need to update the text, it’s totally fine too.

**Comment by hannahkane** [comment]
I think there's been an effort to unify footers across all Mofo properties. I know it's been requested by legal (at least the parts that concern legal), but I think it might also be desirable from a design perspective. @lovegushwa - do you have thoughts on that?

@ScottDowne - are you planning to use mofo-ui in this project? If not, maybe this isn't the right time to unify the footer. Given the tight timeline, let's do whatever is most efficient.

**Comment by ScottDowne** [comment]
I'm mostly done the footer. I'm not directly using mofo-ui for this one because of the time, and because the design for this page is a bit different, I just pasted in the bits I needed, and then started to modify it.

**Comment by hannahkane** [comment]
If the screenshot Sara posted is from https://www.changecopyright.org/, I'm going to suggest we *don't* use the same info for the Contact Us link. It's a mailto for makerparty@, which doesn't seem right for this campaign. I suggest we use the [Mozilla Contact form](https://www.mozilla.org/contact/), as on the Advocacy site, unless there is an email address specifically for this campaign (or for the Advocacy program as a whole?)

**Comment by hannahkane** [comment]
Thinking about "Join the conversation" linking to the Mozilla twitter account—there's a disconnect between the CTA and the destination. I think it should be either:
 * Keep "Join the conversation" but link to a hashtag like #fixcopyright 
 * Keep the Mozilla twitter handle, but change the CTA to "Follow"

Thoughts? @shaghdoosti @lovegushwa @beccaklam

**Comment by shaghdoosti** [comment]
Hey - 

I think changing the contact us link to the mozilla contact form makes sense. 

On the second question - my preference would be to keep the twitter handle and change the CTA to follow. @HPaulJohnson does that sound right to you?

**Comment by ScottDowne** [comment]
We have this footer as it was in the comps: https://changecopyright-org-staging.herokuapp.com/en-US/#get-involved

We also have almost all links, just waiting on share progress. We can close this once we have share progress, or file another issue just for that and close this?

Anything else we want done?

**Comment by hannahkane** [comment]
Per comment above, I think we should change "Join the Conversation" to "Follow" and replace the icon with the Twitter bird. As is, I'd think it was a forum or something.

**Comment by TheoChevalier** [comment]
I will update the string as per https://github.com/MozillaFoundation/Advocacy/issues/723#issuecomment-283081247, if anyone has any objection to that, please speak up before it’s exposed to localizers :)

**Comment by lovegushwa** [comment]
Let's update with "Follow" and the twitter icon @ScottDowne ... can you implement this and close this ticket?

**Comment by TheoChevalier** [comment]
String changed to "Follow" https://github.com/mozilla/copyright/commit/c74083b0277bf70ca9d4302c4c87737e22a1f208#diff-77164019e2bf354a14041bd43ccb5d40L45

**Comment by ScottDowne** [comment]
Also updated the footer icon for both open data and copyright repos.

**Comment by ScottDowne** [comment]
Given that we have this issue https://github.com/MozillaFoundation/Advocacy/issues/743 for just the share link bits, I say we close this.

Feel free to reopen if you disagree or I missed something.
